src/Diagnostics.Scripts/CompilationService/CustomCodeAnalyzers/BlockCallsAnalyzer.cs

```diff
@@ -106,7 +106,9 @@ private void HandleInvocation(OperationAnalysisContext operationContext)
                         _blockConfig.GetMatchingPInvokeConfig(targetPInvokeDll)?.ForEach(piConfig => {
                             if (piConfig.MethodsToBlock?.Count > 0)
                             {
-                                BlockConfig.GetMatchingBlockMessageList(piConfig.MethodsToBlock, invocation.TargetMethod.Name)?.ForEach(strMessage =>
+                                string pInvokeFunctionName = !string.IsNullOrWhiteSpace(invocation.TargetMethod.GetDllImportData().EntryPointName)? invocation.TargetMethod.GetDllImportData().EntryPointName : invocation.TargetMethod.Name;
```

Comment:
nit: you could write line like this: string pInvokeFunctionName = invocation.TargetMethod.GetDllImportData().EntryPointName ?? invocation.TargetMethod.Name;

Reply:
@michimune won't ?? check only for NULL and not for whitespace/empty string?

Reply:
Sure, you are correct. I am going to dismiss my comment.